components/ui/doorhanger/src/main/java/mozilla/components/ui/doorhanger/DoorhangerPrompt.kt

```diff
@@ -0,0 +1,226 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.ui.doorhanger
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.drawable.Drawable
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.RadioButton
+import android.widget.RadioGroup
+import android.widget.TextView
+import mozilla.components.support.ktx.android.view.forEach
+
+/**
+ * Builder for creating a prompt [Doorhanger] providing a way to present decisions to users.
+ *
+ * @param title Title text for this prompt.
```

Comment:
Since the properties are `private` I thought I only need to document the constructor `param`eters. Just looked at the generated docs and the constructor is documented with its parameters. Is there anything we get from using `@property` if they are private?

Reply:
I couldn't find any difference between using one or another, when they are private. I just noticed that in other cases we used `@property`  in constructors.
I think if we are getting the same result is not a problem :)

https://github.com/mozilla-mobile/android-components/blob/1b4ad4a9f18b093bb1e9013d1707ee573869b8c3/components/feature/intent/src/main/java/mozilla/components/feature/intent/IntentProcessor.kt#L28